docs/border-color.md

```diff
@@ -52,43 +51,52 @@ Control the opacity of an element’s border color using the color opacity modif
 -->
 
 ### Individual sides
-
 Use the `border-{l|r|t|b}-{transparent|inherit|current}` or `s-border-{l|r|t|b}-{semantic color}` utilities to set the border color for only one side of an element.
 
 <container>
   <div class="grid grid-cols-4 gap-16 justify-items-center">
-    <div class="h-80 w-80 border-4 rounded-16 s-border s-border-l-positive"></div>
-    <div class="h-80 w-80 border-4 rounded-16 s-border s-border-r-positive"></div>
-    <div class="h-80 w-80 border-4 rounded-16 s-border s-border-t-positive"></div>
-    <div class="h-80 w-80 border-4 rounded-16 s-border s-border-b-positive"></div>
+    <div class="h-80 w-80 border-4 rounded-16 s-border-l-positive"></div>
+    <div class="h-80 w-80 border-4 rounded-16 s-border-r-positive"></div>
+    <div class="h-80 w-80 border-4 rounded-16 s-border-t-positive"></div>
+    <div class="h-80 w-80 border-4 rounded-16 s-border-b-positive"></div>
   </div>
 </container>
 
 ```html
-<div class="s-border-subtle s-border-l-positive ..."></div>
-<div class="s-border s-border-r-positive ..."></div>
-<div class="s-border s-border-t-positive ..."></div>
-<div class="s-border s-border-b-positive ..."></div>
+<div class="s-border-l-positive border-4 ..."></div>
+<div class="s-border-r-positive border-4 ..."></div>
+<div class="s-border-t-positive border-4 ..."></div>
+<div class="s-border-b-positive border-4 ..."></div>
 ```
 
 ### Horizontal and vertical sides
-
 Use the `border-{x|y}-{transparent|inherit|current}` or `s-border-{x|y}-{semantic color}` utilities to set the border color on two sides of an element at the same time.
 
 <container>
   <div class="grid grid-cols-2 gap-16 justify-items-center">
-    <div class="h-80 w-80 border-4 rounded-16 s-border s-border-x-positive"></div>
-    <div class="h-80 w-80 border-4 rounded-16 s-border s-border-y-positive"></div>
+    <div class="h-80 w-80 border-4 rounded-16 s-border-x-positive"></div>
+    <div class="h-80 w-80 border-4 rounded-16 s-border-y-positive"></div>
    </div>
 </container>
 
 ```html
-<div class="s-border s-border-x-positive ..."></div>
-<div class="s-border s-border-y-positive ..."></div>
+<div class="s-border-x-positive border-4 ..."></div>
+<div class="s-border-y-positive border-4 ..."></div>
 ```
 
-### Hover, focus and other states
+### Arbitrary values
+If you need to use a one-off `border-{color}` value, use square brackets to generate a property on the fly using any arbitrary value.
+However, we recommend only using our semantic color tokens.
+This also works with or without an optional direction.
+`border-{x|y|l|r|t|b}-[{any color value or css var}]`
 
+```html
+<div class="border-[--w-s-color-icon]">
+  <!-- ... -->
+</div>
+```
+
```

Comment:
Is there a reason for moving the arbitrary values section to be before Hover, focus and other states? For the other css props that we support arbitrary values, we always have the arbitrary values section at the bottom of the page. :)

Reply:
The arbitrary value docs is actually relevant to the page, in this case "Border color". The other ones are just "extras", which I personally think should live else-where as general info, rather than on every single page? 😄 🤷